docs/form_helper.rst

```diff
@@ -135,22 +135,28 @@ Bootstrap 4 Helper attributes
 All previous, ``bootstrap`` (version 2 and 3) attributes are also settable in bootstrap 4 template pack ``FormHelpers``. Here are listed the ones, that are only available in ``bootstrap4`` template pack:
 
 **use_custom_control = True**
-    Enables the `optional UI customization <https://getbootstrap.com/docs/4.3/components/forms/#custom-forms>`_ of the template pack for radio, checkbox and file field (non clearable only). Useful when you already have customization based on the default interpretation of the template pack. Setting to ``False`` results in the `standard bootstrap <https://getbootstrap.com/docs/4.3/components/forms/#checkboxes-and-radios>`_ classes being applied for radio and checkbox, and Django rendering for file field. See table below for examples. 
+    Enables the `optional UI customization <https://getbootstrap.com/docs/4.3/components/forms/#custom-forms>`_ of the template pack for radio, checkbox, select and file field. Useful when you already have customization based on the default interpretation of the template pack. Setting to ``False`` results in the `standard bootstrap <https://getbootstrap.com/docs/4.3/components/forms/#checkboxes-and-radios>`_ classes being applied for radio and checkbox, and Django rendering for file field. See table below for examples.
     
     The file field requires `additional JS <https://www.w3schools.com/bootstrap4/bootstrap_forms_custom.asp#myFile>`_ to enable its functionality, it is provided within the template pack as vanilla JS. 
     
     Defaults to ``True``.
 
+.. |radio| image:: images/radio.png
+   :width: 300px
+   :align: middle
+.. |customradio| image:: images/custom_radio.png
+   :width: 300px
+   :align: middle
 .. |checkbox| image:: images/checkbox.png
    :width: 300px
    :align: middle
 .. |customcheckbox| image:: images/custom_checkbox.png
    :width: 300px
    :align: middle
-.. |radio| image:: images/radio.png    
+.. |select| image:: images/select.png
    :width: 300px
    :align: middle
-.. |customradio| image:: images/custom_radio.png
+.. |customselect| image:: images/custom_select.png
```

Comment:
```
/home/pi/Projects/django-crispy-forms/docs/form_helper.rst:156: WARNING: image file not readable: images/select.png
/home/pi/Projects/django-crispy-forms/docs/form_helper.rst:159: WARNING: image file not readable: images/custom_select.png
```

I can't seem to build the docs?

Reply:
I'll amend this (I missed adding some files). How are you building the docs? I can't see in the documentation or in the .github action.

Reply:
```
cd docs
make html
```
I might add this in the docs (or read them better if it's there)